### PR TITLE
Update supported chains generation

### LINKED ISSUE #126: Improve exporting supported chains
**Acknowledgements**
- Exporting supported chains should be readible

**To-Do**
- Replace chain ids with chain aliases

### PR DESCRIPTION
**Pull request by hiletmis**
Closes #126

### CODE REVIEW

**Comment by hiletmis** [comment]
@mcoetzee `dapi-management` package calculates supported chains and write them into `supported-chains.json` file as chain ids array.

To simplify review processes chain ids replaced with chain aliases using `chains` package.

File: src/index.ts [hunk 1]
```diff
@@ -3,4 +3,16 @@ import supportedChains from './generated/supported-chains.json';
 import * as api3Chains from '@api3/chains';
 import * as api3Integrations from '@api3/api-integrations';
 
-export { dapis, supportedChains, api3Chains, api3Integrations };
+function supportedChainIds() {
```
Comment (mcoetzee) [inline src/index.ts:6]: Minor: Should we perhaps call this function `getSupportedChainIds`?

**Review by mcoetzee — APPROVED** [review]
Just got one minor comment, so I'm approving so long 👍

File: src/index.ts (hunk 1 above)
Comment (hiletmis) [reply src/index.ts]: Thanks updated function name as suggested.

Comment (hiletmis) [reply src/index.ts]: But changing the name may break previously usages, idk just wondering if its the case

Comment (mcoetzee) [reply src/index.ts]: Previous usages would already be affected though, because the `supportedChains` array was changed to a function.

**Pull request merged**